Dockerfile

```diff
@@ -28,5 +28,8 @@ RUN cd /pip && \
 	--no-deps -r requirements/docker.txt && \
     rm -r build cache
 
+# Need uglify for code minimization.
+RUN npm install
```

Comment:
Yeah, I was wondering about that too. There aren't actually any node packages defined in the container at this point, so I'm surprised the command doesn't just fail.